routes/confirmation/confirmation.spec.js

```diff
@@ -13,37 +14,59 @@ describe('Test confirmation urls', () => {
 })
 
 describe('Test /review', () => {
+  const session = require('supertest-session')
+
+  function extractCsrfToken(res) {
+    var $ = cheerio.load(res.text)
+    return $('[name=_csrf]').val()
+  }
```

Comment:
Let's put this into a generic file so that we don't copy+paste it everywhere.

Maybe `/utils/index.js`, or a `/routes/utils.js` file — not too bothered where.